react-hot-button-component/src/index.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+import hotButton from 'hot-button';
+
+let counter = 1;
+
+class CustomButton extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {isClicked: false};

```

Comment:
Your property names in state should match the data they will hold. Instead of naming it `isClicked` something like `count` or `clickCount` would be better.